docs/smart-contracts/capabilities.md

```diff
@@ -292,6 +292,110 @@ You can test scoped signatures using the `env-sigs` REPL-only function as follow
   (accounts.pay "alice" "bob" 10.0))
 ```
 
+## Use module administrator privileges for capabilities
+
+In Pact 5, you can bring a capability into scope at the top level of a module if you own the administrative privileges to control the module. 
+
+The following example illustrates defining a GOVERNANCE capability for the `test` module to always succeed:
+```pact
+(module test GOVERNANCE
+  (defcap GOVERNANCE() true)
+  (defcap EXAMPLE() true)
+
+    (defun test()
+       (+ 1 3)
+    )
+)
+```
+
+Because the GOVERNANCE capability owns the administrative privileges for the module, you can bring any capability into scope.
+The following example illustrates using the `test.GOVERNANCE` capability in the other-test module:
+
+```pact
+(module other-test G
+    (defcap G() true)
+    (defun test()
+      (with-capability (test.EXAMPLE) (+ 1 1))
```

Comment:
See my comment about this, but any function trying to acquire another's capabilities via `with-capability` is a pretty advanced use case and only works when you own module admin. 

Frankly we haven't seen any legitimate use cases for this yet, so this type of code should be in some sort of advanced section. This is not a type of pattern we want to encourage, as requiring admin powers for modules lets you do whatever you want.... including editing things like tables.